### PR TITLE
Media

### PR DESCRIPTION
**Pull request by bwarner**

### CODE REVIEW

File: lib/media/media.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
```
Comment (hafabenny) [inline lib/media/media.js]: what is classNames used for?  moving away from emotion cx?

**Comment by hafabenny** [comment]
other than the new classNames component question.  looks good! you are cranking these components out!

**Comment by hafabenny** [comment]
is the code coverage a blocker for merge?

**Comment by bwarner** [comment]
No

**Pull request merged**